src/Raven.Server/RavenServer.cs

```diff
@@ -270,8 +272,25 @@ void ConfigureKestrel(KestrelServerOptions options)
                     }
                 }
 
+                // TODO [ppekrol]
```

Comment:
hoho

Reply:
left it here, but I'm not sure if this should exposed via settings or not, going with defaults now